app/src/main/java/ch/epfl/sweng/studdybuddy/firebase/FirebaseReference.java

```diff
@@ -23,11 +23,22 @@
 
     //Reference to firebase
     private final DatabaseReference ref;
+    private static boolean persistenceEnabled = false;
```

Comment:
should this need (static) setters/getters? or is it hardcoded? not sure of the design choices, but this definitely works